src/main/kotlin/net/developerden/devdenbot/pasting/CodeBlockMessageListener.kt

```diff
@@ -46,6 +46,12 @@ class CodeBlockMessageListener @Inject constructor(
         if (!rawText.contains(codeBlock)) {
             return
         }
+        if (rawText.contains(tokenRegex)) {
+            event.message.delete()
+            event.channel.sendMessage("The previous message has been deleted due to").await()
+            event.channel.sendMessage("it containing a discord bot token!").await()
```

Comment:
I think we should make it clear that the bot token should also be changed.
Once it has been sent, it should be considered compromised.